cpp/proto/arcticc/pb2/storage.proto

```diff
@@ -12,6 +12,12 @@ package arcticc.pb2.storage_pb2;
 import "google/protobuf/any.proto";
 import "arcticc/pb2/utils.proto";
 
+enum AWSAuthMethod {
```

Comment:
Why is this here? We had agreed there would be no changes to the proto definitions whatsoever?

Reply:
It's a relic of previous implementation. It's been removed